File: src/impl/RunConfExporter.java
package impl;

import com.intellij.openapi.components.ServiceManager;
import com.intellij.openapi.project.Project;
import org.jetbrains.annotations.NotNull;

public interface RunConfExporter {
	static RunConfExporter getInstance(@NotNull Project project) {
		return ServiceManager.getService(project, RunConfExporter.class);
	}

	void exportConfigs(String pathToFolder);
}

File: src/impl/RunConfExporter.kt
package impl

import com.intellij.openapi.components.ServiceManager
import com.intellij.openapi.project.Project

interface RunConfExporter {

    fun exportConfigs(pathToFolder: String)

    companion object {
        fun getInstance(project: Project): RunConfExporter {
            return ServiceManager.getService(project, RunConfExporter::class.java)
        }
    }
}
